tests/test_DiscreetORM.spec.ts

```diff
@@ -1,11 +1,65 @@
 import * as DiscreetORM from './../clientlib/src/DiscreetORM'
 import { expect } from 'chai';
 import 'mocha';
+import { stringify } from 'querystring';
+import { create } from 'domain';
 
-describe('empty test bench', () => {
-    it('empty test', () => {
-        const result = null;
-        expect(result).to.equal(null);
+function spaceIndependentEquals(s1 : string, s2: string) : boolean{
```

Comment:
Generally would prefer a bit more clear control flow here, but everything looks like it ought to work